release-notes/release-notes59.md

```diff
@@ -216,10 +216,15 @@ The Java server-side components have been updated to the following versions:
 
 ### New Server-side component features
 
-All the services now include a `ignore-port` port option to allow ports specified in `allowed-origins` to be enforced. For details, see: [Configure server-side components: `allowed-origins.ignore-port` (optional)]({{site.baseurl}}/enterprise/server/configure/#allowed-originsignore-portoptional).
+The Spelling service now supports a broader range of dictionaries by introducing support for [Hunspell dictionaries](https://hunspell.github.io/).
```

Comment:
Ah, thank you. This was one of my tasks (release notes). Now I don't have to do it :)